h/browser/chrome/lib/install.js

```diff
@@ -14,7 +14,18 @@ var browserExtension = new HypothesisChromeExtension({
 });
 
 browserExtension.listen(window);
+
 chrome.runtime.onInstalled.addListener(onInstalled);
+
+chrome.runtime.onMessageExternal.addListener(
+  /** Respond to messages sent by the JavaScript from https://hpt.is.
```

Comment:
This indentation is confusing. It should just be an inline comment above the 'addListener' line.